nsqd/nsqd.go

```diff
@@ -80,9 +88,16 @@ func New(opts *Options) *NSQD {
 		notifyChan:           make(chan interface{}),
 		optsNotificationChan: make(chan struct{}, 1),
 		ci:                   clusterinfo.New(opts.Logger, http_api.NewClient(nil)),
+		dl:                   dirlock.New(dataPath),
 	}
 	n.swapOpts(opts)
 
+	err := n.dl.Lock()
+	if err != nil {
+		n.logf("FATAL: --data-path=%s in use", dataPath)
```

Comment:
I think it would be helpful if this error gave a little bit of a hint that it's in use by another nsqd, or that the resolution is to use a different/unique dirpath. 

Thoughts?